curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64e340302bd28513f3e73740.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 64e340302bd28513f3e73740
+title: Step 48
+challengeType: 20
+dashedName: step-48
+---
+
+# --description--
+
+Now, change the comment below the `print()` call into `display our progress`.
+
+# --hints--
+
+You should change `# display starting configuration` into `# display our progress`.
+
+```js
+({ test: () => assert.match(code, /\)\s+#\s*display\sour\sprogress\s+(?=print\()/) })
+```
+
+# --seed--
+
+## --seed-contents--
+
+```py
+NUMBER_OF_DISKS = 4
+rods = {
+    'A': list(range(NUMBER_OF_DISKS, 0, -1)),
+    'B': [],
+    'C': []
+}
+
+--fcc-editable-region--
+def move(n, source, auxiliary, target):
+    if n > 0:
+        # move n - 1 disks from source to auxiliary, so they are out of the way
+        move(n - 1, source, target, auxiliary)
```

Comment:
The previous seed has `auxiliary, target)`, and does not ask Campers to change the order. Which should it be?

Reply:
I pushed a commit changing the test (step-47), because that is what allows Campers to pass. However, until this is fixed, step-48 seed will fail the test.

Reply:
It should be `move(n - 1, source,  auxiliary, target)` int his step